lmp/build.sh

```diff
@@ -33,9 +33,13 @@ cat /root/.gitconfig >>  /home/builder/.gitconfig
 cp /root/.netrc /home/builder/.netrc || true
 
 # Detect the base LmP version we are building on
+# A user may have place arbitrary tags into their lmp-manifest and we don't
+# want to accidentally pick one of them up. Git doesn't really distinguish
+# were tags come from, so:
+git --git-dir .repo/manifests.git tag | xargs git --git-dir .repo/manifests.git tag -d > /dev/null
 run git --git-dir .repo/manifests.git remote add upstream https://github.com/foundriesio/lmp-manifest
 run git --git-dir .repo/manifests.git fetch --tags upstream
-export LMP_VER=$(git --git-dir .repo/manifests.git describe upstream/main --tags --abbrev=0)
+export LMP_VER=$(git --git-dir .repo/manifests.git describe HEAD --tags --abbrev=0)
```

Comment:
We can restore the costumer repo at the end
```
run git --git-dir .repo/manifests.git fetch --tags origin
run git --git-dir .repo/manifests.git  remote upstream
```

Reply:
good catch. I did a force push that moves things to a helper function and I wrote a unit test.